docs/releases/v2.0.0.md

```diff
@@ -83,17 +70,39 @@ kustomize build katalog/prometheus-operated | kubectl apply -f - --server-side
 ```
 
 - `alertmanager-operated`
+
+Alert Manager now reads the notification enpoint for Slack and Healthchecks.io from secrets, instead of having it embedded in the configuration.
```

Comment:
```suggestion
Alertmanager now reads the notification enpoint for Slack and Healthchecks.io from secrets, instead of having it embedded in the configuration.
```